src/utils/getStoredUser.tsx

```diff
@@ -0,0 +1,6 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export default async function getStoredUser() {
+  const me = await AsyncStorage.getItem('me');
+  return me != null ? JSON.parse(me) : null;
```

Comment:
Since you're not using strict equals, could you do `me ? ... : ...` instead of `me != null ? ... : ...`?

Reply:
done